src/users/users.controller.spec.ts

```diff
@@ -250,11 +256,16 @@ describe('UsersController', () => {
           userId: user.id,
           type: EventType.PULL_REQUEST_MERGED,
         });
-
         await eventsJobsController.updateLatestPoints({
           userId: user.id,
           type: EventType.BUG_CAUGHT,
         });
+        // refresh points
```

Comment:
This update needs to be here since I changed from using individual categories to using POOL4